configs/components/openssl.rb

```diff
@@ -13,7 +13,7 @@
     # needed for the makedepend command
     pkg.build_requires 'imake' if platform.name =~ /^el/
     pkg.build_requires 'xorg-x11-util-devel' if platform.name =~ /^sles/
-    pkg.build_requires 'xutils-dev' if platform.is_huaweios?
+    pkg.build_requires 'xutils-dev' if platform.is_deb?
```

Comment:
I'm actually not sure how openssl could have built without the makedepend command, which is in the xutils-dev package for all Debian-based distros. I'll run a test build for each of those architectures just to make sure nothing's been broken.